frontend/src/layout/nav.vue

```diff
@@ -31,39 +31,61 @@
           </template>
           <div :style="{width:200+'px'}">
             <b-dropdown-item to="/queue">The Q</b-dropdown-item>
-            <!--
+
+            <template  v-if="user.role && user.role.role_code=='GA'">
+              <b-dropdown-item @click="clickGAScreen">
+                <b-check :checked="showGAScreenModal"><span style="font-weight: 400;">Show GA Panel</span></b-check>
+              </b-dropdown-item>
+              <b-dropdown-divider />
+            </template>
+
             <b-dropdown-item to="/exams">Manage Exams</b-dropdown-item>
```

Comment:
Let's take out exams and bookings from this PR. We want to be able to merge this change into master tomorrow